src/engine.go

```diff
@@ -161,19 +161,26 @@ func (e *engine) print() string {
 		if !*e.env.getArgs().Eval {
 			break
 		}
-		// escape double quotes contained in the prompt
-		prompt := fmt.Sprintf("PS1=\"%s\"", strings.ReplaceAll(e.string(), "\"", "\"\""))
-		prompt += fmt.Sprintf("\nRPROMPT=\"%s\"", e.rprompt)
+
+		lprompt := e.ansi.escapeText(e.string())
```

Comment:
why do we still need this?

Reply:
No reason, since everything is done before. I missed it in the previous refactoring.